cmd/up/up.go

```diff
@@ -114,17 +114,22 @@ func Up() *cobra.Command {
 			}
 			manifestOpts := contextCMD.ManifestOptions{Filename: upOptions.DevPath, Namespace: upOptions.Namespace, K8sContext: upOptions.K8sContext}
 			oktetoManifest, err := contextCMD.LoadManifestWithContext(ctx, manifestOpts)
-			if err != nil && errors.Is(err, oktetoErrors.ErrManifestNotFound) {
-				if !utils.AskIfOktetoInit(upOptions.DevPath) {
+			if err != nil {
+				if err.Error() == fmt.Errorf(oktetoErrors.ErrNotLogged, okteto.CloudURL).Error() {
```

Comment:
minor: can we wrap this into a custom error? :/

Reply:
correct me if Im wrong but I think that if our errors have a custom format at the moment this is the only way to compare it. **errors.Is()** will return us false, even if both have same message